dask_sql/physical/rex/core/call.py

```diff
@@ -614,22 +615,62 @@ def __init__(self):
         super().__init__(self.timestampadd)
 
     def timestampadd(self, unit, interval, df: SeriesOrScalar):
-        df = convert_to_datetime(df)
-
-        if unit in {"DAY", "SQL_TSI_DAY"}:
-            return df + np.timedelta64(interval, "D")
+        interval = int(interval)
```

Comment:
Like `date_part`, we'll want to normalize our `unit`'s casing here so that something like `select timestampadd(year, ...) ...` will work:

```suggestion
        unit = unit.upper()
        interval = int(interval)
```